seed/static/seed/js/controllers/new_member_modal_controller.js

```diff
@@ -36,7 +36,11 @@ angular.module('BE.seed.controller.new_member_modal', [])
        * adds a user to the org
        */
       $scope.submit_form = function () {
-        user_service.add($scope.user).then(function () {
+        // make `role` a string 
+        const u = _.cloneDeep($scope.user)
+        u.role = u.role.value
+
+        user_service.add(u).then(function () {
```

Comment:
@adrian-lara I found a bug in using the v3 api for user creation, this fixes that.

Reply:
User creation seems to work as expected. Just curious, what was the bug? and was it caused by v3 changes?

Reply:
The role on the FE is actually an object, which v2 used to handle by checking if it was a dict:
https://github.com/SEED-platform/seed/blob/f1d2347d146c9d4164752d8d699966f619464a96/seed/views/users.py#L218
This check was dropped in v3 api (which I think makes sense). So now we always expect role to just be a string. Creating users from the admin page would work previously b/c no `role` property was added to the payload.

Reply:
👍 Thanks for the clarification!